pkg/metrics/metrics.go

```diff
@@ -0,0 +1,123 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	// Registry-namespaces of operator sources.
+	REDHAT_OPERATOR_NS    = "redhat-operators"
+	COMMUNITY_OPERATOR_NS = "community-operators"
+	CERTIFIED_OPERATOR_NS = "certified-operators"
+
+	// Default values which are to be used as labels
+	// for prometheus metrics.
+	REGISTRY_NAMESPACE = "registry_ns"
```

Comment:
What is `REGISTRY_NAMESPACE` used for?

Reply:
It is a label used, for Prometheus metrics.